waspc/ChangeLog.md

```diff
@@ -26,6 +26,8 @@ Remember to check out the [migration guide](https://wasp.sh/docs/migration-guide
 
 ### 🐞 Bug fixes
 
+- Fixed terminal output on Windows: now it is unstyled on Windows, since ansi codes are often not shown correctly on Win shells.
+  This default behaviour can be explicitly controlled via `WASP_TERMINAL_STYLING=0|1` env var. ([#3403](https://github.com/wasp-lang/wasp/pull/3403))
```

Comment:
We don't usually have multiline items in the changelog. Maybe actually noting this behaviour in the documentation is better that in the changelog.